### ISSUE TITLE
Can display aliases instead of filenames on the backlink tab and outlink tab?

### ISSUE THREAD

**Issue by shaopeng1993** [muhammadv-i/obsidian-frontmatter-alias-display#3]
https://github.com/basham/obsidian-alias-from-heading/issues/5

Can we support working with the obsidian-alias-from-heading in the future? I think this is a very good idea~

**Comment by muhammadv-i** [comment]
Good idea! I'll look into it and see what I can do. It's gonna take a while tho because I'm kinda busy for the foreseeable future.
